problemsets-mat223/modules/module1.tex

```diff
@@ -479,7 +479,24 @@
 	Let $\vec x=\xhat-\yhat$, $\vec y=3\xhat-\yhat$, and $\vec r=2\xhat+2\yhat$. Is
 	$\vec r$ a linear combination of $\vec x$ and $\vec y$?
 
-	XXX Finish
+	By definition, $\vec r$ is a linear combination of $\vec x$ and $\vec y$ if there exist scalars $a,b\in\R$ such that
+	\begin{align*}
+	    \vec r&=a\vec x+b\vec y\\
+	   \implies 2\xhat+2\yhat&=a(\xhat-\yhat)+b(3\xhat-\yhat)\\
```

Comment:
Don't use `\implies`. Write another sentence, `Substituting for $\vec x$ and $\vec y$, we see this is equivalent to finding scalars $a,b$ so that ...`